content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,285 @@
+.. image:: calls_for_tenders/calls-for-tenders-blank-quotation.png
    :align: center
    :alt: Blank alternative quotation with alternatives in breadcrumbs.
 
-This causes a :guilabel:`Compose Email` pop-up window to appear, wherein the message to the vendor
-can be customized. Once ready, click :guilabel:`Send` to send an email to the vendor listed on the
-purchase order form.
-
-From this newest purchase order form, click the :guilabel:`Alternatives` tab. Under this tab, all
-three purchase orders can be seen in the :guilabel:`Reference` column. Additionally, the vendors
-are listed under the :guilabel:`Vendor` column, and the order :guilabel:`Total` and
-:guilabel:`Status` of the orders are in the rows, as well.
+This opens a :guilabel:`Compose Email` pop-up window, wherein the message to the vendor can be
+customized. Once ready, click :guilabel:`Send`.
 
-Link a new :abbr:`RFQ (Request for Quotation)` to existing quotations
-=====================================================================
+From this newest form, click the :guilabel:`Alternatives` tab. Under this tab, all three |POs| can
+be seen in the :guilabel:`Reference` column. Additionally, the vendors are listed under the
+:guilabel:`Vendor` column, and the order :guilabel:`Total` and :guilabel:`Status` of the orders are
+in the rows, as well.
 
-Creating alternative quotations directly from a purchase order form under the
-:guilabel:`Alternatives` tab is the easiest way to create and link quotations. However, separate
-:abbr:`RFQs (Requests for Quotation)` can also be linked *after* the fact, even if they are created
-completely separately at first.
+Link a new |RfQ| to existing quotations
+=======================================
 
-To create a new :abbr:`RFQ (Request for Quotation)`, navigate to the :menuselection:`Purchase` app,
-and click :guilabel:`New`.
+Even if a quotation is not created directly from the :guilabel:`Alternatives` tab of another |RfQ|,
+it can still be linked to existing |RfQs|.
 
-Then, add information to the :abbr:`RFQ (Request for Quotation)` form: add a vendor from the
-drop-down next to the :guilabel:`Vendor` field, and click :guilabel:`Add a product` to select a
-product from the drop-down menu in the :guilabel:`Product` column. Then, set the desired purchase
-quantity in the :guilabel:`Quantity` column, and change the purchase price in the
-:guilabel:`Unit Price` column, if desired.
+Begin with creating a new |RfQ|, by navigating to :menuselection:`Purchase app --> New`. Fill out
+the |RfQ| according to the earlier instructions.
 
-Once ready, click :guilabel:`Send by Email`. This causes a :guilabel:`Compose Email` pop-up window
-to appear, wherein the message to the vendor can be customized. Once ready, click :guilabel:`Send`
-to send an email to the vendor listed on the purchase order form.
+Then, once ready, click the :guilabel:`Alternatives` tab. Since this new |PO| was created
```

Comment:
RfQ not a PO, right?

Just trying to make sure things are as clear as possible for the reader to follow along with.  So, if an RfQ is 'technically' a PO, just in its early stages, it still might be beneficial to keep that same, super-accurate terminology in tact, so as to avoid any potential confusion.

Reply:
You are correct. Made the change to reflect that -Suggestion implemented ✅